js/index.js

```diff
@@ -1,42 +1,49 @@
 // ITERATION 1
-
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  let price = product.querySelector(".price span");
```

Comment:
Will any of the variables in this function be reassigned? If not, would "const" be a better variable?